docs/machine-learning/resources/transforms.md

```diff
@@ -1,168 +1,102 @@
 
-## Preprocessing
-| Transform | Definition |
-| --- | --- |
-| <xref:Microsoft.ML.Transforms.BootstrapSamplingTransformer> | Approximates bootstrap sampling using Poisson sampling. |
-| <xref:Microsoft.ML.Transforms.Projections.RandomFourierFeaturizingTransformer> | Produces random Fourier feature. |
-| <xref:Microsoft.ML.Transforms.Text.TokenizingByCharactersTransformer> | Character-oriented tokenizer where text is considered a sequence of characters. |
-| <xref:Microsoft.ML.Transforms.Projections.VectorWhiteningTransformer> | Simplifies optimization to assist with identifying weights. |
-
-## Row Filters
+## Categorical data transformations
 
 | Transform | Definition |
 | --- | --- |
-| <xref:Microsoft.ML.Transforms.RowShufflingTransformer> | Shuffles a randomized cursor attempt to perform using a pool of a given number of rows.  |
-| <xref:Microsoft.ML.Transforms.SkipFilter> | Allows limiting input to a subset of rows by skipping a number of rows. |
-| <xref:Microsoft.ML.Transforms.SkipTakeFilter> | Allows limiting input to a subset of rows at an optional offset. Can be used to implement data paging. When created with SkipTakeFilter.SkipArguments behaves as `SkipFilter`.
-| <xref:Microsoft.ML.Transforms.TakeFilter> | Allows limiting input to a subset of rows by taking N first rows. |
-
-## Schema
+| <xref:Microsoft.ML.CategoricalCatalog.OneHotEncoding*> | Convert one or more text columns into one-hot encoded vectors |
+| <xref:Microsoft.ML.CategoricalCatalog.OneHotHashEncoding*> | Convert one more text columns into hash-based one-hot encoded vectors |
 
-| Transform | Definition |
-| --- | --- |
-| <xref:Microsoft.ML.Transforms.ColumnCopyingTransformer> | Duplicates columns from the dataset.|
-| <xref:Microsoft.ML.Transforms.ColumnSelectingTransformer> | Selects a set of columns to drop or keep from a given input. |
-| <xref:Microsoft.ML.Transforms.FeatureSelection.SlotsDroppingTransformer> | Drops slots from columns.|
-| <xref:Microsoft.ML.Transforms.OptionalColumnTransform> | Creates a new column with the specified type and default values. |
-| <xref:Microsoft.ML.Transforms.RangeFilter> | Filters a dataview on a column of type Single, Double or Key (contiguous). Keeps the values that are in the specified min/max range. NaNs are always filtered out. If the input is a Key type, the min/max are considered percentages of the number of values. |
-
-## TensorFlow
+## Missing values
 
 | Transform | Definition |
 | --- | --- |
-| <xref:Microsoft.ML.Transforms.TensorFlowTransform> | Either scores with pretrained TensorFlow model or retrains TensorFlow model. |
+| <xref:Microsoft.ML.ExtensionsCatalog.IndicateMissingValues*> | Create a new boolean output column, which is true when the value in the input column is missing |
+| <xref:Microsoft.ML.ExtensionsCatalog.ReplaceMissingValues*> | Create a new output column, whose values are set to a default value if the value is missing from the input column, and the input value otherwise |
```

Comment:
Create a new output column, setting the value of a particular row to true to the default value of that type (e.g. 0 for System.Int), if the value is missing from the input column, or input value otherwise

Reply:
Reworded